dialogue-annotations/src/main/java/com/palantir/dialogue/annotations/Request.java

```diff
@@ -95,4 +95,13 @@
 
         Class<? extends ListParamEncoder<?>> encoder() default DefaultListParamEncoder.class;
     }
+
+    /**
+     * Similar to the Feign QueryMap serialization type. Only supports simple string -> string maps.
+     */
+    @Retention(RetentionPolicy.SOURCE)
+    @Target(ElementType.PARAMETER)
+    @interface QueryMap {
+        Class<? extends MultimapParamEncoder<?>> encoder() default DefaultMultimapParamEncoder.class;
```

Comment:
alternatively we could require a value (no default) and nix the Default since I'm not sure how frequently that will be used. No strong preference though.

Reply:
I do like that this is somewhat self documenting (even if duplicative) since it does show you what the presumed default is.